project/users/management/commands/create_groups.py

```diff
@@ -0,0 +1,36 @@
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import Group, Permission
+from django.contrib.contenttypes.models import ContentType
+from project.partners.models import Partner
+from project.program.models import Activity, Presenter
+
+PERMISSIONS = ['view', 'edit', 'add', 'delete']
+permissions = {'media': ['partner'], 'fr': ['partner'],
+               'experience': ['activity', 'presenter'], 'graphics': [],
+               'venue': [], 'speakers': ['presenter']}
```

Comment:
+activity permission for Speakers' team

Reply:
and ['activity', 'presenter'] for media instead of ['partner']

Reply:
I think that partner should stay, there are communication parnters that maybe media team would like to add.

Reply:
Cool, thanks!

Reply:
> +activity permission for Speakers' team

Cool, thanks!